src/PhpSpec/Formatter/DotFormatter.php

```diff
@@ -61,12 +67,18 @@ public function afterExample(ExampleEvent $event)
                 break;
         }
 
-        if ($eventsCount % 50 === 0) {
+        if ($eventsCount % 50 === 0 || $eventsCount === $this->examplesCount) {
             $length = strlen((string) $this->examplesCount);
             $format = sprintf(' %%%dd / %%%dd', $length, $length);
+
+            if ($eventsCount === $this->examplesCount) {
```

Comment:
This condition is being used twice, probably you could create a method which checks this or save result into a variable with meaning name.